### PR TITLE
doc: add minutes for meeting 18 May 2022

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: meetings/2022-05-18.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      * at mention in issue, discussion etc.
+      * post in OpenJS slack channel (s)
+      * Richard we are probably in as good as a place as we can be
+      * No objections from TSC members on proceeding
+
+### nodejs/TSC
+
+* End-of-Life dates of Node.js 16 and OpenSSL 1.1.1 do not align [#1222](https://github.com/nodejs/TSC/issues/1222)
+  * Richard investigating possibility of using RH OpenSSL
+  * Rich should we decided now ?
```
Comment (Trott) [inline meetings/2022-05-18.md:71]:
```suggestion
  * Rich should we decide now ?
```

File: meetings/2022-05-18.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      * at mention in issue, discussion etc.
+      * post in OpenJS slack channel (s)
+      * Richard we are probably in as good as a place as we can be
+      * No objections from TSC members on proceeding
+
+### nodejs/TSC
+
+* End-of-Life dates of Node.js 16 and OpenSSL 1.1.1 do not align [#1222](https://github.com/nodejs/TSC/issues/1222)
+  * Richard investigating possibility of using RH OpenSSL
+  * Rich should we decide now ?
```
Comment (Trott) [inline meetings/2022-05-18.md:71]:
```suggestion
   * Rich should we decide now?
```

**Review by RaisinTen — APPROVED** [review]

File: meetings/2022-05-18.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  * Richard will look a few more days, unless direction changes will likely result in shortening
+    support.
+  * Plan to confirm decision next week.
+
+* 2022 TSC Chair election - Planning for May 1 ballot. [#1201](https://github.com/nodejs/TSC/issues/1201)
+  * on as FYI, unless there are objections Michael (Chair) and Matteo(Vice-chair) will be confirmed end of day May 20.
+
+## Strategic Initiatives
+
+* New one for which champion is not TSC member, planning for updatres on TSC meeting issue, and invites of
```
Comment (sepehrst) [inline meetings/2022-05-18.md:82]:
```suggestion
* New one for which champion is not TSC member, planning for updates on TSC meeting issue, and invites of
```

File: meetings/2022-05-18.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
```
Comment (sepehrst) [inline meetings/2022-05-18.md:82]: typo **updatres** --> **updates**

File: meetings/2022-05-18.md (hunk 3 above)
Comment (Trott) [inline meetings/2022-05-18.md:82]:
```suggestion
* New one for which champion is not TSC member, planning for updates on TSC meeting issue, and invites of
```

**Pull request merged**